HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -7434,6 +7441,48 @@ Patch "Force PlayStation 5 Controller UI" in "UI" by "Hyper"
     );
 }
 
+Patch "Force Screen Space Global Illumination" in "Graphics" by "WasifBoomz" does
```

Comment:
This code might be better off using the `Reflection` library so this can be set per stage.